CONTRIBUTING.md

```diff
@@ -524,6 +524,12 @@ To get a sense for what changes are considered mass rebuilds, see [previously me
   - [Commit conventions](./nixos/README.md#commit-conventions) for changes to `nixos`.
   - [Commit conventions](./doc/README.md#commit-conventions) for changes to `doc`, the Nixpkgs manual.
 
+- When reverting a commit, use the generated summary line `Revert "..."` and `Reapply "..."`.
+  For repeated reverts and reapply commits, the summary line it may be simplified if it gets too complex.
+  Include an explanation for why the revert is needed in the commit message body.
+  If there are any additional fixes, put the fixes in separate commits.
+  The revert commit should only revert the original change.
```

Comment:
I think it's also fine to have a single commit that reverts some change and also does other stuff to keep the package working, but in that case it should follow the normal commit guidelines and just mention that it includes a revert.

Reply:
I'm not sure how to word this without making this an entire section on a honestly rarely used workflow.

I can change this whole bullet point to just "`Revert "..."` and `Reapply "..."` is permitted", and leave the rest to maintainer judgement, but I don't think we really *need* that fast given that reverts are relatively rare.

Reply:
The idea to treat a revert as a cherry-pick but backwards seems tempting though